explainability-core/src/test/java/org/kie/trustyai/explainability/model/TypeTest.java

```diff
@@ -251,14 +252,22 @@ void testPerturbCompositeFeatureTooManyPerturbations() {
     @ParameterizedTest
     @EnumSource
     void testDrop(Type type) {
-        Value v = new Value(1.0);
+        Value v;
+        if (type == Type.TENSOR) {
```

Comment:
I don't like too much all these "if TENSOR" conditions (here and in the other tests) but I don't have any better idea to be honest 🤔